src/applications/check-in/pages/Failed.jsx

```diff
@@ -1,21 +1,13 @@
 import React from 'react';
-import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+import BackToHome from '../components/BackToHome';
+import Footer from '../components/Footer';
 
 const Failed = () => {
-  const contactNumber = '555-867-5309';
-
   return (
     <div className="vads-l-grid-container vads-u-padding-y--5">
       <h1 tabIndex="-1">Please check in with a staff member.</h1>
```

Comment:
kinda outside scope for this PR, but could you add the following in?

"We were unable to check you in. Please see a staff member to complete your check-in."

(or something to that effect)

Reply:
We are going through the IA content review this week. This is going to be updated during that. @kristenmcconnell will have more details as the week goes on.